internal/tests/tests/ue/service_request_data.go

```diff
@@ -185,12 +186,12 @@ func (t ServiceRequestData) Run(ctx context.Context, env engine.Env) error {
 }
 
 func runServiceRequest(ranUENGAPID int64, pduSessionStatus [16]bool, ue *ue.UE) error {
-	err := ue.SendServiceRequest(ranUENGAPID, pduSessionStatus)
+	err := ue.SendServiceRequest(ranUENGAPID, pduSessionStatus, nasMessage.ServiceTypeData)
 	if err != nil {
 		return fmt.Errorf("could not send Service Request NAS message: %v", err)
 	}
 
-	// fr, err := gnb.WaitForMessage(ngapType.NGAPPDUPresentInitiatingMessage, ngapType.InitiatingMessagePresentInitialContextSetupRequest, 500*time.Millisecond)
+	// fr, err = gnb.WaitForMessage(ngapType.NGAPPDUPresentInitiatingMessage, ngapType.InitiatingMessagePresentInitialContextSetupRequest, 500*time.Millisecond)
```

Comment:
I know you didn't introduce it here but we probably can remove this whole commented out block.